docs/use-case-descriptions/message-validation.md

```diff
@@ -13,25 +13,73 @@ If it fails validation, a response message is sent with ‘rejected’ in the re
-| ISP duration rejected            | The message specifies a ISP duration that is not the agreed-upon common value for the market in which it is used.                                                                                                     |
-| Time zone rejected               | The message specifies a time zone that has a different UTC offset than is the agreed- upon common value for the market.                                                                                               |
-| Invalid congestion point         | Unknown congestion point or the recipient is not active at this congestion point.                                                                                                                                     |
-| Unknown reference <Reference\>  | The message with the sequence where is referred to is unknown.</br>For <Reference\> the concerning reference field name can be filled in (for example FlexRequestSequence or PrognosisSequence).                     |
-| Reference Period mismatch        | The message(s) with the sequence where is referred to contains a different Period.                                                                                                                                    |
-| Reference message expired        | The message that is referred to is expired.                                                                                                                                                                           |
-| Reference message revoked        | The message that is referred to is revoked.                                                                                                                                                                           |
-| ISPs out of bounds               | One or more ISPs are outside the tolerated boundaries: ISPs do not exist.                                                                                                                                             |
-| ISP conflict                     | One or more ISPs are defined more than once, possibly because of an incorrect duration.                                                                                                                               |
-| Period out of bounds             | Period of the message is inappropriate.</br>For example: a FlexRequest with a Period in the past or a settlement item in a FlexSettlement with a Period outside the concerning settlement period.                     |
-| ExpirationDateTime out of bounds | ExpirationDateTime is in the past or exceeds the ISPs in the message.                                                                                                                                                 |
+
+| RejectionReason | Cause of rejection | Applicable to messages |
```

Comment:
I think readability will be improved if you stick to the original formatting? (using a markdown formatter in your editor/ IDE)?

Reply:
I'll try to look into that! Good catch